File: 1117.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
    int vezes, inva;
    double media;
    vezes = inva = media = 0;
    while(true){
        double N;
        cin >> N;
        if(vezes == 2){
            break;
        }
        if(N >= 0 and N <= 10){
            vezes += 1;
            media += N;
        }
        else{
            inva += 1;
        }
    }
    for(int i = 0; i < inva; i++){
        cout << "nota invalida" << "\n";
    }
    media = media / 2;
    cout << fixed << setprecision(2);
    cout << "media = " << media << endl;
}

File: 1117.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        int vezes = 0; double media = 0;
        while(true) {
                double n = input.nextDouble();
                if(vezes == 2) {
                        break;
                }
                if(n >= 0 && n <=10) {
                        vezes += 1;
                        media += n;
                }
                else{
                        System.out.println("nota invalida");
                }
        }
        media = media / 2.0;
        System.out.printf("media = %.2f%n", media);
}
}
